docusaurus.config.js

```diff
@@ -1,71 +1,95 @@
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'The open source hyperconverged infrastructure (HCI) solution for a cloud native world',
-  tagline: '',
-  url: 'https://harvesterhci.io',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
-  organizationName: 'harvester', // Usually your GitHub org/user name.
-  projectName: 'harvester.github.io', // Usually your repo name.
+  title:
+    "The open source hyperconverged infrastructure (HCI) solution for a cloud native world",
+  tagline: "",
+  url: "https://harvesterhci.io",
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/favicon.ico",
+  organizationName: "harvester", // Usually your GitHub org/user name.
+  projectName: "harvester.github.io", // Usually your repo name.
   themeConfig: {
     colorMode: {
       // "light" | "dark"
-      defaultMode: 'light',
+      defaultMode: "light",
 
       // Hides the switch in the navbar
       // Useful if you want to support a single color mode
       disableSwitch: true,
     },
     navbar: {
-      title: '',
+      title: "",
       logo: {
-        alt: 'logo',
-        src: 'img/logo_horizontal.svg',
+        alt: "logo",
+        src: "img/logo_horizontal.svg",
       },
       items: [
         {
-          type: 'doc',
-          docId: 'intro',
-          position: 'right',
-          label: 'Docs',
+          label: "Docs",
+          position: "right",
+          items: [
+            {
+              label: "0.2.0",
+              href: "/docs/intro",
+            },
+            // ... add more versions as items if needed in the future
+            /*
+            example:
+            {
+              label: "docs_example",
+              position: "right",
+              items: [
+                {
+                  label: "0.3.0",
+                  href: "/docs/intro",
+                },
+                {
+                  label: "0.2.0",
+                  href: "/docs/0.2.0/intro",
+                },
+              ],
+            },
+            */
+          ],
         },
         {
-          href: 'https://github.com/harvester/harvester',
-          label: 'GitHub',
-          position: 'right',
+          label: "Github",
+          position: "right",
+          href: "https://github.com/harvester/harvester",
         },
       ],
     },
```

Comment:
These lines are changed. Other lines are marked as changed due to auto-formatter plug-in.